local_repo/roles/validation/vars/main.yml

```diff
@@ -128,16 +129,16 @@ omnia_repo_url_ubuntu_fail_msg: "Failed. Please ensure omnia_repo_url_ubuntu is
 omnia_repo_url_ubuntu_url_gpg_key_fail_msg: "Failed. Each item in omnia_repo_url_ubuntu should have 'url' and 'gpgkey' keys defined"
 omnia_registry_fail_msg: "Failed. omnia_registry should be defined in local_repo_config.yml."
 omnia_registry_string_fail_msg: "Each item in omnia_registry should be a string."
-rpm_dir_path: "{{ repo_store_path }}/cluster/{{ cluster_os_type }}/{{ cluster_os_version }}/rpm"
-deb_dir_path: "{{ repo_store_path }}/cluster/{{ cluster_os_type }}/{{ cluster_os_version }}/deb"
+rpm_dir_path: "{{ repo_store_path }}/omnia/offline_repo/cluster/{{ cluster_os_type }}/{{ cluster_os_version }}/rpm"
+deb_dir_path: "{{ repo_store_path }}/omnia/offline_repo/cluster/{{ cluster_os_type }}/{{ cluster_os_version }}/deb"
 
 # Usage: validate_user_registry.yml
 user_registry_fail_msg: "Failed. Please ensure user_registry is non empty list and
 check if there is any indentation error in input/local_repo_config.yml"
 user_registry_fail_host_cert_path_msg: "Failed. Each item in user_registry should have 'host' and 'cert_path' keys defined"
 time_out: 30
-user_registry_msg: "The following user registries are not reachable: [{{ unreachable_registries | join(', ') }}]. If the user registry is not accessible from the Omnia Infrastructure Manager, Omnia will download all the images for the software listed in software_config.json."   # noqa: yaml[line-length]
-cert_path_failure_msg: "Certificate file path {{ item.item.cert_path }} does not exist on the Omnia Infrastructure Manager for host {{ item.item.host }}. Please verify that correct cert_path is given in input/local_repo_config.yml"  # noqa: yaml[line-length]
+user_registry_msg: "The following user registries are not reachable: [{{ unreachable_registries | join(', ') }}]. If the user registry is not accessible from the control plane, Omnia will download all the images for the software listed in software_config.json."   # noqa: yaml[line-length]
```

Comment:
Revert changes for control plane to Omnia Infrastructure Manager in all occurances